sdk/monitor/azure-monitor-opentelemetry-exporter/tests/statsbeat/test_statsbeat.py

```diff
@@ -534,6 +545,23 @@ def test_get_feature_metric_none(self):
         self.assertEqual(len(observations), 0)
         self.assertEqual(_StatsbeatMetrics._FEATURE_ATTRIBUTES["feature"], 0)
 
+    # pylint: disable=protected-access
+    def test_get_feature_metric_instrumentation(self):
```

Comment:
This is the same test name. Did you mean to have it be something else? It's a negative case.

Reply:
Oops. I removed the previous one.